source/api/image/3.0/change-log.md

```diff
@@ -30,74 +30,103 @@ The following changes are backwards incompatible with version 2.1.1 of the Image
+Per the [deprecation warning][image21-full-dep] in the previous version, the value `full` is no longer allowed for the size parameter, `max` must be used instead.
+
+See issues [#678](https://github.com/IIIF/api/issues/678) and [#1369](https://github.com/IIIF/api/issues/1369).
 
 #### 1.1.2. Change canonical form of size parameter to _`w,h`_
 
-Per the [deprecation warning][image21-full-dep] and the [inconsistency warning][image21-size-inconsistency] about the size parameter in the previous version, the canonical form of the size parameter is now _`w,h`_ unless `max` is requested. This resolves the inconsistency between the server-preferred values in the `sizes` object, and the canonical form of the size parameter. In order to request preferred sizes, a client should use the `width` and `height` values from `sizes` unmodified to build the _`w,h`_ size to request. Clients should also use canonical form of the size parameter _`w,h`_ when constructing tile requests. See issues [#544](https://github.com/IIIF/api/issues/544) and [#678](https://github.com/IIIF/api/issues/678).
+Per the [deprecation warning][image21-full-dep] and the [inconsistency warning][image21-size-inconsistency] about the size parameter in the previous version, the canonical form of the size parameter is now _`w,h`_ unless `max` is requested. This resolves the inconsistency between the server-preferred values in the `sizes` object, and the canonical form of the size parameter. In order to request preferred sizes, a client should use the `width` and `height` values from `sizes` unmodified to build the _`w,h`_ size to request. Clients should also use canonical form of the size parameter _`w,h`_ when constructing tile requests.
+
+See issues [#544](https://github.com/IIIF/api/issues/544) and [#678](https://github.com/IIIF/api/issues/678).
 
-#### 1.1.3. Size parameter value must not scale image to larger than the extracted region
+#### 1.1.3. Changes to upscaling of images to larger than the extracted region
 
-The value of the size parameter must not result in an image larger than the extracted region. Attempts to do so must generate an error response. Previous versions allowed implementations to [optionally support scaling up][image21-size]. The `sizeAboveFull` feature name was also removed. See issues [#693](https://github.com/IIIF/api/issues/693) and [#1370](https://github.com/IIIF/api/issues/1370).
+The value of the size parameter must not result in an image larger than the extracted region, and attempts to do so must generate an error response. Previous versions allowed implementations to [optionally and implicitly support scaling up][image21-size]. The `sizeAboveFull` feature name was also removed. Servers may still support upscaling by indicating that they support the `sizeUpscaling` feature, which requires the size parameter to be prefixed with a caret (`^`).
```

Comment:
Prefix with "Unless prefixed with a caret (`^`), the value of the size parameter..."